libraries/AP_HAL_ChibiOS/hwdef/JHEMCU-GF16-F405/hwdef_pindef.dat

```diff
@@ -0,0 +1,167 @@
+# hw definition file for processing by chibios_pins.py
+# JHEMCU GF16 by V-22
+# With F405 MCU, MPU6000 IMU and MAX7456 series OSD
+# Based on Mamba F405 hwdef from jeanphilippehell
+# thanks to betaflight for pin information (https://github.com/betaflight/unified-targets/blob/master/configs/default/JHEF-JHEF405PRO.config)
+# Board info https://github.com/jhemcu/FC-ESC-Firmware/tree/main/GF16-F405%26ESC
```

Comment:
This will almost exclusively deployed in a BF/X configuration - you should set the FRAME_TYPE to reflect this.

Reply:
```
# Motor order implies Betaflight/X for standard ESCs
define HAL_FRAME_TYPE_DEFAULT 12
```